ipas_default.config

```diff
@@ -0,0 +1,410 @@
+
```

Comment:
Is it possible to move this to some other folder instead of root?

Reply:
Rather than having this config in the root or somewhere else, would it be possible to move this `pyproject.toml` file?

We ideally want to collect all the configuration within `pyproject.toml` file. I can see from the bandit [documentation](https://bandit.readthedocs.io/en/latest/config.html), it seems to be possible: 

```
[tool.bandit]
exclude_dirs = ["tests", "path/to/file"]
tests = ["B201", "B301"]
skips = ["B101", "B601"]

[tool.bandit.any_other_function_with_shell_equals_true]
no_shell = [
  "os.execl",
  ...
```

And to run bandir based on the configurations from `pyproject.toml` file

```
bandit -c pyproject.toml -r .
```

Reply:
yes, it can. can you point out the proper target?

Reply:
@samet-akcay Yes, I know that but my concern is on that I've copied the bandit config file from IPAS (Intel Product Assurance and Security) which contains minimum requirements for specific task in the SDL (internal release process), so I think, it would be better to keep it as same as original to track the updating their side and applying them to our projects efficiently. how do you think?

Reply:
here is the internal repo which has configuration: https://github.com/intel-innersource/applications.security.bandit-config